src/UnicodePlots.jl

```diff
@@ -50,7 +49,9 @@ export
     densityplot, densityplot!,
     heatmap,
     spy,
-    boxplot, boxplot!
+    boxplot, boxplot!,
+
+    savefig
```

Comment:
Will this conflict with other packages ?

Reply:
Potentially, but I'd say we should strive for a common terminology rather than introduce slightly different terms with the same meaning to avoid conflicts

Reply:
So you say it's OK to export savefig from `UnicodePlots` ?

Reply:
I'd say so.